runtime/src/main/java/org/corfudb/runtime/CheckpointWriter.java

```diff
@@ -207,6 +207,22 @@ private long nonCachedAppend(Object object, UUID ... streamIDs) {
         return sv.append(object, null, CacheOption.WRITE_AROUND, streamIDs);
     }
 
+    /*
+     * Pack smr entries into one checkpoint entry
+     */
+    private void convertToCheckpointEntry(MultiSMREntry smrEntries) {
```

Comment:
At the first glimpse I saw the name `convertToCheckpointEntry`, I thought it should have a return value. I have to look inside the function to realize that it performs both conversion and appending of a CheckpointEntry. So perhaps rename it to `convertAndAppendCheckpointEntry` or just `appendCheckpointEntry` since append should already include the step of conversion. Also update the comment to reflect this.

Reply:
Good suggestion. Will make that change